html/components/_tooltip.scss

```diff
@@ -1,11 +1,10 @@
 ////
 /// @group tooltip
 ////
-
 .sprk-c-Tooltip__container {
   position: relative;
   display: inline-flex;
-  vertical-align: middle;
+  vertical-align: baseline;
```

Comment:
I wonder if it needs vertical-align at all. I tried getting rid of it and it looks the same. Do you think there's a need to be explicit about it?